packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -67,10 +68,18 @@ function currentGitInfo() {
 	}
 }
 
+function validateCategories(categories) {
+	if (!(categories.length === new Set(categories).size)) throw new Error('Repeated categories are not allowed');
```

Comment:
It's good you've added a check for this but shouldn't that be `categories.length !== new Set(categories).size)`?

Reply:
Yes, fixed it.